patterns/posts-list.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ * Title: List of posts
```

Comment:
We are introducing a string here so we might need to make this not be a pattern and update on a potential point release

Reply:
I removed it